.eslintrc.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     },
     overrides: [
         {
-            files: ['*.spec.ts'],
+            files: ['*.spec.ts', 'test-*.ts'],
```

Comment:
As tests are all now collected in a `test` directory, perhaps something like this?
```suggestion
            files: ['src/test/**'],
```

Reply:
Thanks for catching that! Will update with upcoming commit.